1. Building a To-Do List Without Framework/submission/index.js

```diff
@@ -104,22 +105,28 @@ const saveTodo = (e) => {
 };
 
 const clearTodo = () => {
-    const clearInfo = `
+        if (todoList.length){
+            const approveClear = confirm("Are you sure you want to delete your all items");
+            if(approveClear){
+                `${localStorage.removeItem("todoList")}
+                ${(todoList = [])}
+                ${renderList()}`
```

Comment:
There is no need to call functions inside template strings unless you are storing it in a variable. 
And these statements do not return anything so it is of no use to call it in a template string

Reply:
Okay got it.